frontend-react/.env.test

```diff
@@ -1,13 +1,13 @@
-;; this is set by env var being set `CLIENT_ENV: test` in the github action
+; this is set by env var being set `CLIENT_ENV: test` in the github action
 REACT_APP_TITLE=CDC Prime ReportStream TEST
 REACT_APP_DESCRIPTION=ReportStream Dashboard Application
-REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+REACT_APP_ICON=https://test.reportstream.cdc.gov//assets/cdc-logo.svg
 
-REACT_APP_BASE_URL=http://localhost:3000
-REACT_APP_BACKEND_URL=http://localhost:3000
-; REACT_APP_BACKEND_URL=http://localhost:8090 ????
+REACT_APP_BASE_URL=https://test.reportstream.gov
+REACT_APP_BACKEND_URL=https://test.prime.cdc.gov
```

Comment:
I think this is overriding `.env.development` on local causing the test failure

Reply:
MSW configs now use the designated backend url per env file, so no tests should fail, even if the wrong env file is prioritized for a certain environment!

Reply:
Think the changes you made to the test are working? Can we resolve this?